src/components/Molecules/story.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {useRef, useEffect}from "react";
+import Component from "./index";
+
+export default {
+  component: Component,
+  title: "Molecules/Tasks",
+  parameters: {
+    backgrounds: {
+      default: "dark",
+    },
+  },
+};
+
+const Template = (args) => <Component {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  onTaskChange : (Name) => {
+    console.log(Name);
+  },
```

Comment:
```suggestion
  onTaskChange : (name) => {
    console.log(name);
  },
```
通常の変数はローワーキャメルケースでお願いします。